lib/internal/perf/resource_timing.js

```diff
@@ -190,7 +206,15 @@ ObjectDefineProperties(PerformanceResourceTiming.prototype, {
   },
 });
 
-function createPerformanceResourceTiming(requestedUrl, initiatorType, timingInfo, cacheMode = '') {
+function createPerformanceResourceTiming(
+  requestedUrl,
+  initiatorType,
+  timingInfo,
+  cacheMode = '',
+  bodyInfo,
```

Comment:
bodyInfo is currently unused because there's a possible bug where the resource timing is taking fields like encodedBodySize/decodedBodySize from the resource's timing info rather than its resource info.